docs/01-start-here/03-how-to-deploy.md

```diff
@@ -0,0 +1,34 @@
+# How to deploy
+
+Your pull request is ready, all comments made by reviewers have been addressed and CI results (Teamcity) are green.
+Congratulations, you can deploy your code to production. Here are the steps to do so:
+
+##Deploying
+
+1. Merge your pull request to master
+2. Wait for the tests and the build to finish (Teamcity)
+3. Once the build is done, it would be automatically deployed to CODE (The Guardian staging environment)
+4. Verify that your change works properly on CODE
+5. You can find out who is also currently deploying by using the [Deploy Radiator](https://frontend.gutools.co.uk/deploys-radiator)
+6. If several people are pushing to PROD at the same time, please coordinates with each other. Slack channel #dotcom-push is a good place to do so. 
+7. When ready use `goo` tool to deploy the build to production. (ex: `./goo deploy --prod -b BUILD_NUMBER` and then confirm). You will need valid AWS credentials. Please use [Janus](https://janus.gutools.co.uk/#) to obtain them.
```

Comment:
We can deploy the latest without the `-b BUILD_NUMBER` but do we want to teach people to always put build number?